modules/battery_manager/ajax/add_entry.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Battery Manager adder
+ *
+ * Checks Test Battery for duplicates and handles insertions into Test Battery
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  Loris license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+
+// Determine which action is called
+if (isset($_GET['action'])) {
+    $action = $_GET['action'];
+    if ($action == "checkForDuplicate") {
+        echo checkForDuplicate();
```

Comment:
curiousity, why does this line have an `echo` while line 23 doesnt ?

Reply:
because checkForDuplicate must return a JSON representation of the duplicate entry to the JS file (or null if there isn't one)